tests/unit/test_network_connections.py

```diff
@@ -215,7 +215,9 @@ def setUp(self):
         }
 
     def assertValidationError(self, v, value):
-        self.assertRaises(ValidationError, v.validate, value)
+        self.assertRaises(
+            network_lsr.argument_validator.ValidationError, v.validate, value
+        )
```

Comment:
If others agree with this change, please put it into its own commit/PR, this seems to be a valid change that could be merged - also the other, similar lines. :+1:

Reply:
Is this change due to a CodeQL error?  If so, did CodeQL suggest this as the correction?

Reply:
CodeQL complains about duplicate import statements: `Module 'network_lsr.argument_validator' is imported with both 'import' and 'import from'.` And no, CodeQL did not give us suggestions.

Reply:
https://github.com/linux-system-roles/network/security/code-scanning/13 has the suggestion to do

```
import network_lsr
ValidationError = network_lsr.argument_validator.ValidationError
```

to avoid importing it twice.